tftrt/examples/image-classification/Colab-TF20-TF-TRT-inference-from-Keras-saved-model.ipynb

```diff
@@ -0,0 +1,1118 @@
GQsJBLNf9yJ3fP/78L+EW12GdisPu\nZqqqi6onAAe6yvWXjVEfNG6+59P83N/+No/WRzCXO6kWsgyVGmgBQvbv8eqpQuSWPX/CV0+spBd6\nEI4SL7iQOoDThGttYn91hPue+Cus9x7WjE2hOKrKM9tTUgh4W4v0hVotu0OaCl7KLNba1K/zc0pW\nfinyzGSgpigczhXUvdC8fo9o1sF0ZdPocwIm2VNEhLGxMWJq5mLPdXn9PlTzTiuNl0IfSyOo/g7x\n0mbAcBbomb+cfwwJPt0iyjMZNzmT/H9/o6P/87NBEZzoYKD8TL/bf2+WGtPn51j670hL69mGcwW1\nz6KevUPzfPTDdyBFyR2P7OJ97/87fvmnfpqP/M17OXHocV594xTt7a/lf+z6I9SacSYvYDUxpWzP\n2RilJ7JXSTKjkCJHUB6OVZ8ieqNud9hVP8KKtRfzkx/6b1yyuWAtW2hfOMad3VO8rir50kOf4tHD\n/4D3HvHS2ECERgU7N0TMEt7BhdtfTKTHFZdDmcbpUJCsSzLDCaycVLZu386e+Yoj879LjD2KIrJ6\nbDPOCSK5ESQYKq1mvjdiNB1fEUJcIKVEnTqkFKhDL0eyGMTstewLSNTE2EOiIZaXARShV9XUqUfM\nq/NZHiym/POn4IZzigSBZR/QM+FsXdt+t/PbCbm7q0/7uvrSWmIMRBGWP85QxCFvh8RlOoxDAv72\nev/OBXzpcwf42Ae/SnnyMF+6ax+7H3mC3/v/fp0P/vmfs+dLH+KdP3Q9fv153HcCOuUL+MVf/Tiz\nndxMiTGClfQalfO6F6hIhJCQ0PQZUv5baoQbN72OiWIjvg4cPPgAx47t5zc/9jMUFew93uVrpx4i\nHu9wkYMXXvV9bN26kx3rXgZRiUHwCMmK7CKnkFLuxh6Tz9MGFnQlL36jMtV6FWrk1Ux19Ho9tvlL\n6XaPccnma1gs1zIXbwbpsWNyK5MFlK4N4pvNpkhIQqEtkDFijHSrhEobSLT8GCnli3XpxrMoiDos\nZgVtUpPNOJdFZn3eDmkVZVbMdjkgKIoCXyi+iANR5TPhnGiM9MU9oR/p9dk51wdKJ7i869X80vJm\nwZAA86jN0sTu609w+wTw9NeF09/Pfg1jaVUtNsfQdygUEZ5qIeOpaoaqWXgVy6/nST7ES/6fGgmv\nwchFMzspKojLZjun31kkq2rHeE6cDt9W+PQtX6SshZ+9915esLXkh37qrTh7IyEaVgipjpx/cA8P\nHPkar7zvB0g/+3qOHFiBswReEIu42EWkjXMBkuFKl5tgfXk0HMkZ12y+mr1HvkLlFDfXo9s+wiKz\nbN4k9JzhFuHGB+H67/tRzAInZ+fZkIzPPPEBblj3tlx/05g1lxKItVm9doIjh2rmHnkR29Y8xNhK\nIcYCo6blJujFDmVrgqp7LTMTH2GFdVgsKsr2GtTvZe/DDzBvkcXQZtxrs9DgEamo6xptvIIlRZIF\nvGvTC/0RmIpCI1qUxFjjyzZ11aMo8rosZiieEHqAkjRraRoVMXpSqnCulWcH3T8DEuwP8PY7wksJ\n5Uzk8CQBAYZkapaeNjU+0yjO2WTpvx7Iaat9gzTzmTwOy4eslx7X03WKl5KpiAy2aJYGk5n0lKoK\nOPWNp0t/3vKfZ130XIU+fDt+2w4unix42w++HSzigOiFWOe1r9f+1Ft5TVTsIx/l1Y/u5uOvfysq\nORI0EZKMEawLKC7lD71zjuiFJIqziizzVvD48f0U60o2XH4Fpa3i/DUX4Q4e5rcP3c9/e+HPc/gl\ns/S6EXXK6qlN1CeMGza9Dk1CHcE30VWgg2jBrNzPiTpxWXUXt58KTMaIc7GZ0e1AMnppHu/bWPUa\nWP8AY70x6FSsqI9zzD3MiqkWIQQqugCI1XhVfOEIIW+thJRwYtShh9MxjHow/iUhIlLQqxbwXun2\nOrRbY4QQCNah5ceoqgQWEYNuL1H6bAlqqd9EOeeltIY4k1LMuSDy8K3E1zM7ufR3c2dv+L20NIpe\n8j3nHLFKzYK9jJLhbwJ+5nd/ESOndjFkEug0VpZOGpfBaKyeDYi0iLNdkk9YFUnmsBBAEiVCrZBc\nnjk0M1xMmOZJgDpEvCbowcart3Pfl6bAWpw6cZw4Dpctwlfu/APWXvd2nBiWPOmkYUmpgwddoJCC\nikCvqnBacP5l2zl4KqLcytZXfBef/fXfphxbQUyCM8V7IRQglRKcYVWBdDZT2Cy0TnD41pVcdM2L\neMgeoJws0UogGeoUS4kkCV9AHQLSjMMU2so2oJI9g3xZZqk4USQJySLeFXkGEAZ1cOeEjsKUK7OX\ndkrEus7ma7J8KuJ0nBM1wX7Ul9xQViBKnq9yjaPb0kPtj7QAxCTUsWmnD3hCc6h8lgHJp26CLN0C\nOf1+DPxGluJsndkzvc7+7fSudN9cKafWaSDaMIgMpUl7vw7h0/4f3Czvhy518+tbkg6OvWmUjOw2\nvznoxYpuXQ0IUNUjzmNhOLOpYrSPAeefh9bCptjJXrkaG6KEaEsagKZ5LIoECXzwGAW1BaYX1vLx\nm3+DMVaQ3AI9O0IdSv73l/1vrH/RO9CYDY+QyGOHPpVrZ1Lya599L2aOGKDwEyCJvQfv5MjxPWyx\njaxrKb36Adb4C7IIBBVVL+TIyypcqvGSmD25lnLqfFaWq5i45DqOL+xgds4xXmkz15uwVBCS4sUT\nzeEFnI94UerUAxJl2QZVQoA6WiOPn1BxFKXPGoFiIL5xSuxShkQ98B1yzXvmwArGyrGz/o3OCSmt\nEUYYYYRvFc6JSHCEEUYY4VuFEQmOMMIIz2mMSHCEEUZ4TmNEgiOMMMJzGiMSHGGEEZ7TGJHgCCOM\n8JzGiARHGGGE5zRGJDjCCCM8pzEiwRFGGOE5jREJjjDCCM9pjEhwhBFGeE5jRIIjjDDCcxojEhxh\nhBGe0
```

Comment:
Any particular reason `max_workspace_size_bytes=4000000000` is not set to 8GB similar to other precisions?

Reply:
I was trying to prevent INT8 section crashes, which probably relates to memory. Now i'm forcing Colab to restart before INT8 section

Reply:
Make it 8Gb now after instruction users to restart colab, which will prevent INT8 section crashes.